plugins/UM3NetworkPrinting/tests/Cloud/TestCloudOutputDeviceManager.py

```diff
@@ -55,7 +56,9 @@ def _beforeTearDown(self):
         devices = self.device_manager.getOutputDevices()
         # TODO: Check active device
 
-        response_clusters = self.clusters_response.get("data", [])
+        response_clusters = []
```

Comment:
This could be simplified to 

```python
response_clusters = [CloudClusterResponse(**cluster).toDict() for cluster in self.clusters_response.get("data", [])]
```

Reply:
That's not really simpler is it? Same number of variables and function calls... It's just putting it all on one line which makes the line harder to read at a glance.

Reply:
Whatever you prefer. Your code, your party.

Reply:
I appreciate the feedback but too often I find myself reading a whole for each loop in a single line and going, "Wait... heh?... ah... ok."

Reply:
Sure thing. List comprehensions give better performance in Python, but probably in this case wouldn't make a noticeable difference.